src/content/1.7/modules/sell/techvalidation-keysteps.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Technical validation - Key steps
+---
+
+# Technical Validation : Key steps
+
+Have you ever wondered how the PrestaShop Addons Team treats the validation ?
```

Comment:
Not sure if `treats` is the correct translation 🤔

"performs" ? "considers" ? "carries out" ?

Reply:
Well, I would write _Have you ever wondered how the PrestaShop Addons Team proceeds with the validation?_, what do you think?